pom.xml

```diff
@@ -463,6 +463,7 @@
 
                 <module>hazelcast</module>
                 <module>helidon</module>
+				<module>hexagonal-architecture</module>
```

Comment:
I see that the main pom.xml already contains a mix of tabs and spaces, around this part of the code it's all spaces. But around line 1204 it's all tabs. What would you prefer me to do here?